src/nvim/ex_getln.c

```diff
@@ -2550,6 +2550,7 @@ static bool cmdpreview_may_show(CommandLineState *s)
     goto end;
   }
 
+  cursorcmd();
```

Comment:
Should probably add a comment above referencing this PR just in case someone wonders why this line exists in the future